spec/models/zipped_moab_version_spec.rb

```diff
@@ -45,4 +44,18 @@
   it { is_expected.to have_db_index(:preserved_copy_id) }
   it { is_expected.to have_db_index(:status) }
 
+  describe '#replicate!' do
+    before { zmv.save! }
+
+    it 'if PC is unreplicatable, returns false, does not enqueue' do
+      expect(pc).to receive(:replicatable_status?).and_return(false)
+      expect(ZipmakerJob).not_to receive(:perform_later)
+      expect(zmv.replicate!).to be(false)
+    end
+    it 'if PC is replicatable, passes druid and version to Zipmaker' do
+      expect(pc).to receive(:replicatable_status?).and_return(true)
+      expect(ZipmakerJob).to receive(:perform_later).with(pc.preserved_object.druid, pc.version)
+      zmv.replicate!
```

Comment:
total nitpick, and wouldn't hold up merge over this, but the slight difference between these two tests makes me think that `ZMV#replicate!` should return true after its call to `perform_later`, just so it has a consistent return type.  unless the return value of `perform_later` would be useful (in which case i'd advocate having the guard clause return `nil`, just because i like consistent/narrow return types).

like i said, a nitpick, and not something i'd hold this up over.

_edit_:  ok, after a bit of looking, it seems that `perform_later` is supposed to return the job id, which does seem useful?  if that's the case, i'd advocate (mildly) for the guard clause returning `nil`.

Reply:
apparently i also missed that the return types were listed on the method comment.

anyway, posted #1020 to deal with this nitpick, so as to get this PR merged.